ibis/pandas/execution/window.py

```diff
@@ -30,12 +30,19 @@
 from ibis.pandas.execution import util
 
 
-def _post_process_empty(scalar, parent, order_by, group_by):
+def _post_process_empty(result, parent, order_by, group_by):
```

Comment:
can you type these args (and return type of this function)

Reply:
Done